pkg/controllers/kubeconfig/suite_test.go

```diff
@@ -23,6 +24,7 @@ var (
 	cfg     *rest.Config
 	cl      client.Client
 	ctx     = context.Background()
+	timeout = 10 * time.Minute
```

Comment:
10 minute timeout?

Reply:
probably it's too much, I replaced it with 1 minute

Reply:
Normally I would expect to use something like 10 seconds, 1 minute for the controller to react would indicate an issue to me and surely means the watches aren't working right

Reply:
👍done